ansible/roles/apache-application-lavarel5/files/lavarel5.conf

```diff
@@ -0,0 +1,8 @@
+DocumentRoot ${baseRootDir}/public
+DirectoryIndex index.php
+
+<Directory ${baseRootDir}/public>
```

Comment:
Do you mean the line 5 `Options FollowSymLinks Indexes`? I have absolutely no idea thus I never used lavarel. I just added it for all vhosts as a default.